cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/SpUtils.kt

```diff
@@ -37,8 +40,27 @@ fun generateCcpaUspString(
             else -> "N"
         }
         val lspaCoveredTransaction = if (signedLspa == null || signedLspa == false) "N" else "Y"
-        "$specificationVersion$opportunityToOptOut$optOutSale$lspaCoveredTransaction"
-    } else DEFAULT_CCPA_USP_STRING
+        val usPString = "$specificationVersion$opportunityToOptOut$optOutSale$lspaCoveredTransaction"
+        logger?.computation(
```

Comment:
@carmelo-iriti these loggers are really getting on the way of readability. Can we get rid of it?

Reply:
Unfortunately no, because I use them to check if the conditions are matched and in production they are not logging anything (using the veriants technique)